script.js

```diff
@@ -34,9 +34,24 @@ function clearValidation() {
         })
 }
 
-function dataToForm(id) {
-    const value = $$("table").getItem(id);
-    $$("form").setValues(value);
+function randomInteger(min, max) {
+    let rand = min + Math.random() * (max + 1 - min);
+    return Math.floor(rand);
+}
+
+function addRandomUser() {
+    const countries = ["Germany", "USA", "Canada", "Russia", "China", "France", "Italy", "Spain"];
+    const names = ["John", "Bill", "Jacob", "William", "Liam", "Michael", "Alexander", "Noah"];
+    // const newUser = {
+    //     name: names[randomInteger(0, countries.length - 1)],
+    //     country: countries[randomInteger(0, names.length - 1)],
+    //     age: randomInteger(1, 100)
+    //}
+    $$("user_list").add({
```

Comment:
if there are many items, new ones are added to the end of data, so that users do not see them. try to:

- either add new items to the beginning
- or scroll list by API to show the new item

(same for datatable actually)

Reply:
Added second parameter to Add method. Now it adds users to the beginning of the List.